assets/templates/partials/calendar/items/subscription-links.tmpl

```diff
@@ -35,7 +35,10 @@
           alt=""
         >
       </span>
-      <a href="#" class="ons-list__link ons-u-td-no ons-u-mr-no">
+      <a
+        href="https://www.ons.gov.uk/calendar/releasecalendar"
```

Comment:
We don't normally include `prod` domains. Can you adjust to having the path only?

```suggestion
        href="/calendar/releasecalendar"
```